### PR TITLE
Impose terminology guidelines

### PR DESCRIPTION
**Pull request by moxiegirl**
Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@SvenDowideit The link check seems to be reporting errors on `draft` docs.  Is this expected behavior.

**Comment by SvenDowideit** [comment]
yeah, expected, but just plain not-nice - maybe i can make a change :)

File: docs/docker-introduction.md [hunk 1]
```diff
@@ -113,11 +113,11 @@ Docker Engine should be installed and running in your machine. To verify that Do
         This message shows that your installation appears to be working correctly.
 
         To generate this message, Docker took the following steps:
-         1. The Docker client contacted the Docker daemon.
-         2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
-         3. The Docker daemon created a new container from that image which runs the
+         1. The Docker Engine CLI client contacted the Docker Engine daemon.
```
Comment (SvenDowideit) [inline docs/docker-introduction.md]: `Docker Engine daemon` sounds hokey :/

File: docs/linux/step_four.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ when the image is loaded.
 1. Now, build your new image by typing the `docker build -t docker-whale .` command in your terminal (don't forget the . period).
 
         $ docker build -t docker-whale .
-        Sending build context to Docker daemon 158.8 MB
+        Sending build context to Docker Engine daemon 158.8 MB
```
Comment (SvenDowideit) [inline docs/linux/step_four.md]: really? they changed the cmdline output?

File: docs/mac/index.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ identifier = "mac_getstarted"
 
 This getting started is for non-technical users who are interested in learning about Docker. By following this getting started, you'll learn fundamental Docker features by performing some simple tasks. You'll learn how to:
 
-* install Docker
+* install Docker software using Docker Toolbox
```
Comment (SvenDowideit) [inline docs/mac/index.md]: install _the_ ? mmm, no, me language but goodley

File: docs/mac/index.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ The getting started was user tested to reduce the chance of users having problem
 
 ### Make sure you understand...
 
-This getting started uses Docker commands with a terminal window. You don't need
-to be experienced using a command line, but you should be familiar with how to
-open one and type commands.
+This getting started uses Docker Engine CLI commands with a terminal window. You
```
Comment (SvenDowideit) [inline docs/mac/index.md]: in a terminal ?

File: docs/mac/step_five.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ weight = 5
 
 You've built something really cool, you should share it. In this next section,
 you'll do just that. You'll need a Docker Hub account. Then, you'll push your
-image up to it so other people with Docker can run it.
+image up to it so other people with Docker Engine can run it.
```
Comment (SvenDowideit) [inline docs/mac/step_five.md]: ke?

File: docs/mac/step_four.md [hunk 6]
```diff
@@ -108,7 +108,7 @@ when the image is loaded.
 3. Now, build your new image by typing the `docker build -t docker-whale .` command in your terminal (don't forget the . period).
 
         $ docker build -t docker-whale .
-        Sending build context to Docker daemon 158.8 MB
+        Sending build context to Docker Engine daemon 158.8 MB
```
Comment (SvenDowideit) [inline docs/mac/step_four.md]: naw, my Docker ain't say'n at.

File: docs/docker-introduction.md (hunk 1 above)
Comment (moxiegirl) [inline docs/docker-introduction.md]: this a draft file --- doesnt' go in.  I have to pull or rewrite this.  So, I'm gonna not enter this comment. You are welcome to review it later after if it ends up going in.

File: docs/mac/index.md (hunk 3 above)
Comment (moxiegirl) [inline docs/mac/index.md]: the is not needed for Docker software...but could use it for the other.

File: docs/mac/step_five.md (hunk 5 above)
Comment (moxiegirl) [inline docs/mac/step_five.md]: Oh? is there another Docker product that has a daemon capable of running images without the Engine?

**Comment by moxiegirl** [comment]
author LGTM

**Comment by SvenDowideit** [comment]
LGTM - the checks are because the markdownlinter is being over-enthusiastic - 

https://github.com/docker/markdownlint/pull/8 should solve that.

**Pull request merged**